paddle/fluid/compiler/piano/note/note.proto

```diff
@@ -49,8 +51,8 @@ message ShapeProto {
 // Shape of params and res
 message SignatureProto {
   repeated ShapeProto parameters = 1;
-  repeated ShapeProto results = 2;
-  repeated string parameter_names = 3;
+  repeated string parameter_names = 2;
+  optional ShapeProto result = 3;
```

Comment:
I saw you changed the result from `repeated` to `optional`, are we going to support multiple returns? If so, we had better support `repeated` results.

Reply:
Yes, we support multiple return values, and will use a Tuple instruction to pack them even though there only a single return. Here, only one ShapeProto declared on the result, because both the Shape C++ class and proto （I update the ShapeProto on new commit) can contain multiple sub-shape through their tuple_shapes member.